opentelemetry-sdk/tests/trace/test_trace.py

```diff
@@ -201,6 +201,18 @@ def test_span_processor_for_source(self):
             span2.span_processor, tracer_source._active_span_processor
         )
 
+    def test_get_current_span_multiple_tracers(self):
+        """In the case where there are multiple tracers,
+        get_current_span will return the same active span
+        for both tracers.
+        """
+        tracer_1 = new_tracer()
+        tracer_2 = new_tracer()
+        root = tracer_1.start_span("root")
+        with tracer_1.use_span(root, True):
+            self.assertIs(tracer_1.get_current_span(), root)
+            self.assertIs(tracer_2.get_current_span(), root)
+
```

Comment:
```suggestion
self.assertIs(tracer_2.get_current_span(), tracer_2.get_current_span())
```

Might want to check this outside of the context manager too to check that we don't just attach the old span to the tracer on exit.